DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearningContent.cshtml

```diff
@@ -39,11 +40,25 @@
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     <h1 id="page-heading" class="nhsuk-heading-xl">@Model.SectionName</h1>
+  </div>
+</div>
 
-    @{ var contentType = "assessment"; }
-    <partial name="Shared/_JavaScriptDisabledError" model="contentType" />
+@{ var contentType = "assessment"; }
+<partial name="Shared/_JavaScriptDisabledError" model="contentType" />
 
-    <iframe src="@Model.ContentSource" class="nhsuk-grid-column-full nhsuk-u-margin-bottom-6 js-only-block" height="800">
-    </iframe>
-  </div>
+<div class="nhsuk-u-margin-bottom-6 content-viewer_iframe-wrapper js-only-block" id="content-viewer_iframe-wrapper">
```

Comment:
If this is exactly the same for all three content viewers, might be worth pulling out to a partial? 🙂

Reply:
I assume this should contain both the iframes and buttons. What would be a good name for this partial?

Reply:
Good question. Maybe something like `_IFrameWithFullscreen`? 🙂